### PR TITLE
Vinvoor

### PR DESCRIPTION
**Pull request by Topvennie**
- Added `Vinvoor` init.
- Moved some files around.
- Added a `dev.sh`script for easier development.
  - Runs the backend, backend db and frontend
  - Includes hot module reload for both the backend and frontend.

### CODE REVIEW

File: vinvoor/public/vite.svg
Comment (hannes-dev) [inline vinvoor/public/vite.svg:1]: weg ermee

File: vinvoor/src/App.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+}
+
+.card {
+  padding: 2em;
+}
+
+.read-the-docs {
+  color: #888;
+}
```
Comment (hannes-dev) [inline vinvoor/src/App.css:42]: onnodig?

File: vinvoor/src/assets/react.svg
Comment (hannes-dev) [inline vinvoor/src/assets/react.svg:1]: weg ermee

File: vinvoor/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import "./App.css";
+
+function App() {
+    return <h1>Zess Fronted, Coming Soon!</h1>;
```
Comment (hannes-dev) [inline vinvoor/src/App.tsx:4]: fronted

**Comment by Topvennie** [comment]
That's all part of the basic init of a react project

**Pull request closed without merging**